src/coreclr/jit/gentree.cpp

```diff
@@ -31029,7 +31030,9 @@ bool GenTree::IsVectorPerElementMask(var_types simdBaseType, unsigned simdSize)
             case GT_AND:
             case GT_AND_NOT:
             case GT_OR:
+            case GT_OR_NOT:
```

Comment:
why do we need change here? Isn't this just for base instructions?

Reply:
This function is checking that a node is a mask. If this node is `AND/AND_NOT/OR/XOR` and both of it's children are masks, then it is also a mask. `OR_NOT/XOR_NOT` also fit into that pattern. I don't think any hwintrinsics today are going to end up there, but we do have predicate `OR_NOT/XOR_NOT` for SVE, so I think we might end up using it later.

Reply:
got it.